src/ai/backend/manager/cli/image_impl.py

```diff
@@ -192,3 +194,58 @@ async def dealias(cli_ctx, alias):
             log.exception("Alias not found.")
             return
         await session.delete(alias_row)
+
+
+async def validate_image_alias(cli_ctx, alias: str) -> None:
+    async with (
+        connect_database(cli_ctx.local_config) as db,
+        db.begin_session() as session,
+    ):
+        try:
+            image_row = await ImageRow.from_alias(session, alias)
+            if image_row is None:
+                raise UnknownImageReference
+
+            for key, value in validate_image_labels(image_row.labels).items():
+                if key[:11] == "ai.backend.":
+                    print(f"{key[11:]:<30}: ", end="")
+                else:
+                    print(f"{key:<30}: ", end="")
+                print(*value) if isinstance(value, list) else print(value)
+
+        except UnknownImageReference:
+            log.error(f"No images were found with alias: {alias}")
+        except Exception:
+            log.exception("An error occurred.")
```

Comment:
```suggestion
            log.exception(f"An error occurred. (e: {str(e)})")
```
Let's log a little bit more info about the error

Reply:
This is not required as per https://github.com/lablup/backend.ai/pull/1891#pullrequestreview-1887533685